docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -565,20 +563,20 @@ Consider setting property with `ml_train_model_set_property_for_mode()` when dat
 
 #### Set a dataset from file
 
-To create a `dataset` from files. Each file must contains an array of samples.
-A single sample consists of an array of raw float array for input and another array or raw float array for labels which contains same size as model output:
+To create a `dataset` from files. Each file must contain an array of samples.
```

Comment:
If the two sentences are joined, it will adequately make sense. Please let me know what you think. 
```suggestion
To create a `dataset` from files, each file must contain an array of samples.
```